server/src/main/java/org/apache/iotdb/db/engine/memtable/AbstractMemTable.java

```diff
@@ -595,6 +600,11 @@ public boolean checkIfChunkDoesNotExist(IDeviceID deviceId, String measurement)
     return !memChunkGroup.contains(measurement);
   }
 
+  @Override
+  public boolean ifChunkGroupExist(IDeviceID deviceID) {
```

Comment:
```suggestion
  public boolean isChunkGroupExist(IDeviceID deviceID) {
```
Generally, methods that return a boolean are named starting with "is".

Reply:
Thanks ! And this method has been renamed now.